test/sqlancer/mzcompose.py

```diff
@@ -53,13 +57,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
     seed = args.seed or random.randint(0, 2**31)
 
-    def kill_sqlancer_with_delay() -> None:
-        time.sleep(args.runtime)
-        c.kill("sqlancer", signal="SIGINT")
```

Comment:
So there is no longer a need to kill sqllancer by force?

Reply:
Yes, the `--timeout-seconds` works well enough for SQLancer. It doesn't seem to generate problematic queries which would hang for a long time.